src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1365,5 +1362,1533 @@ public void TestNormalizeScopedLocals()
+                    D = 5l,
+                    E = 2.5f,
+                    F = new AndAnotherClass()
+                    {
+                    }
+                  }
+                }
+                """);
+            TestNormalizeExpression(
+                "new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m}}}", """
+                new SomeClass()
+                {
+                  A = 1,
+                  B = 2,
+                  C = new SomeOtherClass()
+                  {
+                    D = 5l,
+                    E = 2.5f,
+                    F = new AndAnotherClass()
+                    {
+                      G = 7u,
+                      H = 3.72m
+                    }
+                  }
+                }
+                """);
+            TestNormalizeExpression(
+                "new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m,}}}", """
+                new SomeClass()
+                {
+                  A = 1,
+                  B = 2,
+                  C = new SomeOtherClass()
+                  {
+                    D = 5l,
+                    E = 2.5f,
+                    F = new AndAnotherClass()
+                    {
+                      G = 7u,
+                      H = 3.72m,
+                    }
+                  }
+                }
+                """);
+            TestNormalizeExpression(
+                "new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m,},}}", """
+                new SomeClass()
+                {
+                  A = 1,
+                  B = 2,
+                  C = new SomeOtherClass()
+                  {
+                    D = 5l,
+                    E = 2.5f,
+                    F = new AndAnotherClass()
+                    {
+                      G = 7u,
+                      H = 3.72m,
+                    },
+                  }
+                }
+                """);
+            TestNormalizeExpression(
+                "new SomeClass(){A=1,B=2,C=new SomeOtherClass(){D=5l,E=2.5f,F=new AndAnotherClass(){G=7u,H=3.72m,},},}", """
+                new SomeClass()
+                {
+                  A = 1,
+                  B = 2,
+                  C = new SomeOtherClass()
+                  {
+                    D = 5l,
+                    E = 2.5f,
+                    F = new AndAnotherClass()
+                    {
+                      G = 7u,
+                      H = 3.72m,
+                    },
+                  },
+                }
+                """);
+        }
+
+        [Fact, WorkItem(61204, "https://github.com/dotnet/roslyn/issues/61204")]
+        public void TestNormalizeObjectInitializer_SingleLineContext()
+        {
+            VerifySingleLineInitializer("new{}", "new { }");
+            VerifySingleLineInitializer("new{A=1,B=2}", "new { A = 1, B = 2 }");
```

Comment:
Can we put each single-line baseline on its own line so it's easier to visually compare the before/after?